src/main/drivers/bus_i2c_hal.c

```diff
@@ -37,6 +37,17 @@
 
 #define CLOCKSPEED 800000    // i2c clockspeed 400kHz default (conform specs), 800kHz  and  1200kHz (Betaflight default)
 
+// Number of bits in I2C protocol phase
+#define LEN_ADDR 7
```

Comment:
(actually you need at least 9 bits (8 databits + ack), it is unknown in which state unstick is called. But after all the effort, it is OK to keep it this way ...)